cicd/3-app/javabuilder/template.yml.erb

```diff
@@ -857,18 +858,19 @@ Resources:
     Properties:
       AlarmName: !Sub "${SubdomainName}_<%=name.downcase%>_high_concurrent_executions"
       AlarmDescription: !Sub |
-          Alarm if javabuilder usage exceeds 400 concurrent
-          executions for 10 minutes. Occasional spikes are expected, but
-          long-running high usage is an indication of an attack. Page the student learning
-          team for further investigation. See this doc for investigation steps
+          Alarm if javabuilder usage has high concurrent executions for 10 minutes.
+          Occasional spikes are expected, but long-running high usage is an indication
+          of an attack. If this is occuring on the demo environment, this is a non-urgent
+          issue as we expect occasional periods of high usage. If it is on production, 
+          page the student learning team for further investigation. See this doc for investigation steps
           https://docs.google.com/document/d/1bHvV6pvUcwxgZpw0YWBmxFggQL5KqYx9zwolwkZhjU8/edit#bookmark=id.xs1gcuxrw6ze
       ActionsEnabled: true
       AlarmActions:
-          - !If [SilenceAlertsCondition, !Ref AWS::NoValue, !Sub "arn:aws:sns:${AWS::Region}:${AWS::AccountId}:CDO-Urgent"]
+          - !If [SilenceAlertsCondition, !Ref AWS::NoValue, !If [IsDemoCondition, !Sub "arn:aws:sns:${AWS::Region}:${AWS::AccountId}:javabuilder-demo-high-concurrency", !Sub "arn:aws:sns:${AWS::Region}:${AWS::AccountId}:CDO-Urgent"]]
```

Comment:
To eliminate `IsDemoCondition`:

**Easy**: Parameterize the `ElevatedSevereErrorRate` SNS topic name and pass it in as a new config option (then follow with one of these better options)

**Medium**: Parameterize all of the SNS topic names, for consistency

**Hard**: Create the SNS topics within this template so you can `!Ref` them directly. You can conditionally create them based on the `SilenceAlertsCondition` and keep the `!If` for that here. Then, we could conditionally _also_ notify CDO-Urgent for some alarms in production (I know, I know, "I thought you wanted to get rid of environment checks?" - I think an IsProd check for the purpose of escalating alerts is fine, and I think this would be a cleaner way to do it)



_*SNS is about 50 cents per 1 million requests, after the first free million. I don't expect this alarm to contribute in any significance to that number, so sending to both the per-stack topic and cdo-urgent isn't a major concern._

Reply:
Thanks! I started a [slack thread](https://codedotorg.slack.com/archives/C03DBDN67B7/p1684949225430359) to figure out the appropriate follow-up tasks here. For now I'm going with the "easy" option, updated and verifying with my dev instance now.

Reply:
tested that things work as expected, so this is ready for review again!